client/my-sites/media-library/content.jsx

```diff
@@ -136,11 +144,19 @@ module.exports = React.createClass( {
 					);
 					break;
 				default:
-					message = this.translate(
```

Comment:
The only other thing I can think of doing would be to update the error response from the server with more information, but likely I'll need to integrate with flux for: https://github.com/Automattic/wp-calypso/pull/4172#issuecomment-199538543 and as for proper re-fetching of storage limits after uploads/deletes are completed